project-testnet.yaml

```diff
@@ -168,22 +168,27 @@ dataSources:
           kind: ethereum/LogHandler
           filter:
             topics:
-              - CreateProject(address indexed creator, uint256 indexed projectId, string projectMetadata, uint8 projectType, bytes32 deploymentId, bytes32 deploymentMetadata)
+              - ProjectCreated(address indexed creator, uint256 indexed projectId, string projectMetadata, uint8 projectType, bytes32 deploymentId, bytes32 deploymentMetadata)
```

Comment:
You might want to keep the original one unless you decide to re-create the testnet.

Reply:
Would prefer to re-create the Testnet, other than keep these compatible codes. Don't think we will have further breaking change before Mainet.

What's your suggestion.